qemu/tests/block_stream_installation.py

```diff
@@ -0,0 +1,36 @@
+import time
+import random
+
+from virttest import utils_test
+from virttest import utils_misc
+
+from qemu.tests import blk_stream
+
+
+def run(test, params, env):
+    """
+    block_stream_installation test:
+    1). guest installation
+    2). live snapshot during guest installation
+    3). block stream afterwards
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    args = (test, params, env)
+    bg = utils_misc.InterruptedThread(utils_test.run_virt_sub_test, args,
```

Comment:
IT is Ok, but Maybe this way:
```
bg = utils_misc.InterruptedThread(utils_test.run_virt_sub_test,
                                         (test, params, env),
                                       {"sub_type": "unattended_install"})
```

Reply:
@hereischen Allow me keep it this way since it looks cleaner for me. Thanks :)